Libraries/Text/TextInput/RCTBaseTextInputView.m

```diff
@@ -413,17 +413,13 @@ - (BOOL)textInputShouldReturn
   // `onSubmitEditing` is called when "Submit" button
   // (the blue key on onscreen keyboard) did pressed
   // (no connection to any specific "submitting" process).
-#if TARGET_OS_OSX // [TODO(macOS Candidate ISS#2710739)
```

Comment:
I'm not seeing this if statement `if (_onBlurSubmit)` upstream- maybe it makes sense to have it on iOS and macOS but if we do that then it should be marked with our GitHub bug tag. Can you confirm we want this check for both iOS and macOS though?

Reply:
Upstream added the `_onBlurSubmit` check for iOS in 521b16730dd07d80261086c2f33eed2a766d404e, but it gets backed out with 7ebb71a32f211a449b40c74f270ed1549254a876. I suppose we could cherry pick the latter.

Reply:
If it was backed out as a breaking change then lets not include it either. So cherry picking the latter makes the most sense to me so we're not in a broken state

Reply:
Done.